src/Oauth1.php

```diff
@@ -257,8 +258,14 @@ private function signUsingRsaSha1($baseString)
                 . 'requires the OpenSSL extension.');
         }
 
+        if (isset($this->config['private_key'])) {
+            $privateKeyData = $this->config['private_key'];
+        } else {
+            $privateKeyData = file_get_contents($this->config['private_key_file']);
+        }
```

Comment:
I think it makes sense to have a check like this: https://github.com/guzzle/oauth-subscriber/pull/55/files#diff-b39c1f44cc7e5e46b562bae636876623R266 WDYT?